text-generation-inference/server/text_generation_server/cli.py

```diff
@@ -0,0 +1,101 @@
+import sys
+from pathlib import Path
+from typing import Optional
+
+import typer
+from loguru import logger
+
+
+app = typer.Typer()
+
+
+@app.command()
+def serve(
+    model_id: str,
+    revision: Optional[str] = None,
+    sharded: bool = False,
+    trust_remote_code: bool = None,
+    uds_path: Path = "/tmp/text-generation-server",
```

Comment:
nit: not really a `Path` here.
```suggestion
    uds_path: Union[Path, str] = "/tmp/text-generation-server",
```